### PR TITLE
add auth and some style

### PR DESCRIPTION
**Pull request by mrnobody3**

### CODE REVIEW

File: src/components/Header/UserMenu/UserMenu.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    navigate("/");
+  };
+
+  return (
+    <>
+      <ButtonGroup
+        variant="contained"
+        aria-label="outlined primary button group"
+      >
+        <Button type="button" onClick={() => navigate("/user")}>
```
Comment (SerhiienkoRuslan) [inline src/components/Header/UserMenu/UserMenu.jsx:21]: use Link/NavLink

File: src/components/RegisterForm/RegisterForm.jsx [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          type="password"
+          autoComplete="current-password"
+          value={password}
+          name="password"
+          onChange={handelChange}
+        />
+        <Button
+          className={s.btn}
+          variant="contained"
+          type="submit"
```
Comment (SerhiienkoRuslan) [inline src/components/RegisterForm/RegisterForm.jsx:94]: Change type="submit" to type="button" and then you can remove  e.preventDefault(); from handleSubmit

Comment (mrnobody3) [reply src/components/RegisterForm/RegisterForm.jsx:94]: it doesn't work with "button"

File: src/components/Header/UserMenu/UserMenu.jsx (hunk 1 above)
Comment (mrnobody3) [reply src/components/Header/UserMenu/UserMenu.jsx:21]: +

**Review by SerhiienkoRuslan — APPROVED** [review]

**Pull request merged**